Core/Authoring/ManifestDependency.cs

```diff
@@ -17,5 +17,8 @@ public class ManifestDependency
 
         [XmlAttribute("exclude")]
         public string Exclude { get; set; }
+
+        [XmlAttribute("include")]
+        public string Include { get; set; }
```

Comment:
guess we are missing "include in the GUI", create a new issue for that?

![image](https://user-images.githubusercontent.com/5808377/33100449-b6e5c206-cf14-11e7-812a-619e64227a1a.png)